Dockerfile

```diff
@@ -2,6 +2,16 @@ FROM registry.access.redhat.com/ubi8/ubi:latest AS build-stage0
 ARG OC_VERSION="stable"
 ENV OC_URL="https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/${OC_VERSION}"
 
+# AWS cli args and env variables
+# Replace AWS client zipfile with specific file to pin to a specific version
+# (eg: "awscli-exe-linux-x86_64-2.0.30.zip")
+ARG AWSCLI_VERSION="awscli-exe-linux-x86_64.zip"
+ENV AWSCLI_URL="https://awscli.amazonaws.com/${AWSCLI_VERSION}"
+ENV AWSSIG_URL="https://awscli.amazonaws.com/${AWSCLI_VERSION}.sig"
```

Comment:
Is the signature used?
I didn't see that in the dockerfile 😳

Reply:
No, it is  a copy paste error on my part, good catch. Fixed now :)